<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterProductInformation</class>
 <widget class="QDialog" name="RegisterProductInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="productInformation">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>등록할 기기 제품 명</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerCancel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerCheck">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>productInformation</tabstop>
  <tabstop>registerCheck</tabstop>
  <tabstop>registerCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
